sleuth_crawler/scraper/scraper/pipelines.py

```diff
@@ -6,7 +6,7 @@
 # See: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
 import json
 from sleuth_backend.solr.solr import add_item
-from scraper.items import GenericPage
+from .items import GenericPage
```

Comment:
should this just be `items` instead of `.items`?

Reply:
Good point, I don't think the `.` needs to be there. I'll remove it.